src/main/java/com/google/sps/ProjectCenter.java

```diff
@@ -99,9 +99,11 @@ public JobModel fetch(String jobId) throws IOException {
         try {
             job = request.execute();
         } catch (GoogleJsonResponseException e) {
+            // If we receive an error Message, we try further other regions
             continue;
         }
 
+        // If the request was successdful
```

Comment:
Another tiny spelling mistake - not sure if these are really changes worth pointing out?? 😅
 
```suggestion
        // If the request was successful
```

Reply:
They are :D Thanks

Reply:
Resolved in https://github.com/googleinterns/step229-2020/pull/15/commits/d93c9abd9592079a818b7ad7396acdd078cc1aab